tools/build.py

```diff
@@ -244,11 +248,19 @@ def configure_jerry(arguments):
 
     return subprocess.call(cmake_cmd)
 
-def make_jerry(arguments, target=None):
-    make_cmd = ['make', '--no-print-directory', '-j', str(arguments.jobs), '-C', arguments.builddir]
+def make_jerry(arguments):
+    if sys.platform == 'win32':
+        make_cmd = ['cmake', '--build', arguments.builddir, '--config', arguments.build_type]
+    else:
+        make_cmd = ['make', '--no-print-directory', '-j', str(arguments.jobs), '-C', arguments.builddir]
 
-    if target:
-        make_cmd.append(target)
+    return subprocess.call(make_cmd)
+
+def install_jerry(arguments):
+    if sys.platform == 'win32':
+        make_cmd = ['cmake', '--install', arguments.builddir, '--config', arguments.build_type]
```

Comment:
Hi Ossy, I've just bumped into that '--install' switch was introduced in Cmake 3.15, it doesn't exist in previous versions (See: https://cmake.org/cmake/help/v3.15/release/3.15.html). I used the VS2019's internal version of CMake (VS2019 was shipped with 3.14). In order to get the built project installed 'cmake ... --target install' could be used in this case.
Do you have any plan for supporting former versions of CMake? Of course, I know JerryScript on Windows is experimental currently, that's why it's just a question. :)

Reply:
I found a workaround, which doesn't need --install option. The latest commit already works with very old cmake too.